simulator/src/main/resources/reference.conf

```diff
@@ -40,75 +40,75 @@ caffeine.simulator {
 
   policies = [
     # Policies that provide an optimal upper bound
-    "opt.Unbounded",
-    "opt.Clairvoyant",
+    #"opt.Unbounded",
+    #"opt.Clairvoyant",
 
     # Policies based on maintaining a linked-list cross-cutting the hash table
-    "linked.Lru",
-    "linked.Mru",
-    "linked.Lfu",
-    "linked.Mfu",
-    "linked.Fifo",
-    "linked.Clock",
-    "linked.S4Lru",
-    "linked.MultiQueue",
-    "linked.SegmentedLru",
+    #"linked.Lru",
+    #"linked.Mru",
+    #"linked.Lfu",
+    #"linked.Mfu",
+    #linked.Fifo",
+    #"linked.Clock",
+    #"linked.S4Lru",
+    #"linked.MultiQueue",
+    #"linked.SegmentedLru",
 
     # Policies based on obtaining a random sampling from the hash table
-    "sampled.Lru",
-    "sampled.Mru",
-    "sampled.Lfu",
-    "sampled.Mfu",
-    "sampled.Fifo",
-    "sampled.Random",
+    #"sampled.Lru",
+    #"sampled.Mru",
+    #"sampled.Lfu",
+    #"sampled.Mfu",
+    #"sampled.Fifo",
+    #"sampled.Random",
 
     # Policies based on the 2Q algorithm
-    "two-queue.TwoQueue",
-    "two-queue.TuQueue",
+    #"two-queue.TwoQueue",
+    #"two-queue.TuQueue",
 
     # Policies based on a sketch algorithm
-    "sketch.WindowTinyLfu",
-    "sketch.S4WindowTinyLfu",
-    "sketch.SimpleWindowTinyLfu",
-    "sketch.RandomWindowTinyLfu",
-    "sketch.FullySegmentedWindowTinyLfu",
+    #"sketch.WindowTinyLfu",
+    #"sketch.S4WindowTinyLfu",
+    #"sketch.SimpleWindowTinyLfu",
+    #"sketch.RandomWindowTinyLfu",
+    #"sketch.FullySegmentedWindowTinyLfu",
 
-    "sketch.AdaptiveTinyLfu",
-    "sketch.AdaptiveWindowTinyLfu",
+    #"sketch.AdaptiveTinyLfu",
+    #"sketch.AdaptiveWindowTinyLfu",
 
-    "sketch.TinyCache",
-    "sketch.TinyCache_GhostCache",
-    "sketch.WindowTinyCache",
+    #"sketch.TinyCache",
+    #"sketch.TinyCache_GhostCache",
+    #"sketch.WindowTinyCache",
 
     # Policies based on the LIRS algorithm
-    "irr.Lirs",
-    "irr.ClockPro",
+    #"irr.Lirs",
+    #"irr.ClockPro",
 
     # Policies based on the ARC algorithm
-    "adaptive.Arc",
-    "adaptive.Car",
-    "adaptive.Cart",
+    #"adaptive.Arc",
+    #"adaptive.Car",
+    #"adaptive.Cart",
 
     # Caching products
-    "product.Guava",
-    "product.TCache",
+    #"product.Guava",
+    #"product.TCache",
     "product.Cache2k",
     "product.Caffeine",
-    "product.Ehcache2",
+    #"product.Ehcache2",
     "product.Ehcache3",
```

Comment:
Ehcache3 is painfully slow for large traces due to O(n) eviction (it naively scans the cache to construct a random sample). You'll want to turn it off most of the time. (It was even worse until 3.1.2 when it was MRU causing a high miss rate).